api/tests/team/integration/domain/usecases/update-membership-last-accessed-at.usecase.test.js

```diff
@@ -0,0 +1,32 @@
+import { Membership } from '../../../../../src/shared/domain/models/Membership.js';
+import { updateMembershipLastAccessedAt } from '../../../../../src/team/domain/usecases/update-membership-last-accessed-at.usecase.js';
+import * as membershipRepository from '../../../../../src/team/infrastructure/repositories/membership.repository.js';
+import { databaseBuilder, expect, knex } from '../../../../test-helper.js';
+
+describe('Integration | Team | UseCases | update-membership-last-accessed-at', function () {
+  it('updates membership last accessed at', async function () {
```

Comment:
Dans le code on va plutôt chercher cette information en tant que « lastAccessedAt » : 
```suggestion
  it('updates membership lastAccessedAt', async function () {
```